infrastructure/docker/services/php/Dockerfile

```diff
@@ -70,14 +69,13 @@ RUN useradd -s /bin/false nginx
 
 COPY frontend/php-configuration /etc/php/${PHP_VERSION}
 COPY frontend/etc/nginx/. /etc/nginx/
-COPY frontend/etc/service/. /etc/service/
 
 RUN phpenmod app-default \
     && phpenmod app-fpm
 
 EXPOSE 80
 
-CMD ["runsvdir", "-P", "/etc/service"]
+CMD ["runsvdir", "-P", "/var/www/infrastructure/docker/services/php/frontend/etc/service"]
```

Comment:
hmm, this is not handy. 

I have 2 symfony apps. But I want to use the vary same image `php`. 

So i used:

```
    pro:
        build:
            context: services/php
            target: web
    back:
        build:
            context: services/php
            target: web
```
But since the volume is mounted on both applications, there is a conflict:

```
root@ab4e7cad923e:/var/www# runsvdir -P /var/www/infrastructure/docker/services/php/web/etc/service
runsv php-fpm: fatal: unable to lock supervise/lock: temporary failure
runsv nginx: fatal: unable to lock supervise/lock: temporary failure
```

IMHO, we should NOT share container state with the host.

Reply:
I fixed like that for now

```diff
--- a/infrastructure/docker/services/php/Dockerfile
+++ b/infrastructure/docker/services/php/Dockerfile
@@ -83,6 +83,9 @@ RUN useradd -s /bin/false nginx
 
 COPY web/php-configuration /etc/php/${PHP_VERSION}
 COPY web/etc/nginx/. /etc/nginx/
+RUN rm -rf /etc/service/
+COPY web/etc/service/. /etc/service/
+RUN chmod 777 /etc/service/*/supervise/
 
 RUN phpenmod app-default \
     && phpenmod app-fpm \
@@ -90,7 +93,7 @@ RUN phpenmod app-default \
 
 EXPOSE 80
 
-CMD ["runsvdir", "-P", "/var/www/infrastructure/docker/services/php/web/etc/service"]
+CMD ["runsvdir", "-P", "/etc/service"]
 
 FROM php-base AS worker
```

But I don't really like it